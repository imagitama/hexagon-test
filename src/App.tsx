import React from 'react'
import { Global } from '@emotion/core'
import globalStyles from './globalStyles'
import CalculatorInput from './components/calculator-input/calculator-input'
import CalculatorOutput from './components/calculator-output/calculator-output'

export default () => (
  <>
    <Global styles={globalStyles} />
    <h1>Number to words calculator</h1>
    <h2>Input</h2>
    <CalculatorInput />
    <h2>Output</h2>
    <CalculatorOutput />
  </>
)
