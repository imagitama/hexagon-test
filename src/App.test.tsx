import React from 'react'
import { shallow, ShallowWrapper } from 'enzyme'
import App from './App'

describe('App', () => {
  describe('When mounted', () => {
    let component: ShallowWrapper

    beforeAll(() => {
      component = shallow(<App />)
    })

    it('Renders correctly', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
