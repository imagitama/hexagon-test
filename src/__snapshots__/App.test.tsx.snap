// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App When mounted Renders correctly 1`] = `
<Fragment>
  <ForwardRef(render)
    styles={
      Object {
        "map": undefined,
        "name": "19v6tzj",
        "next": undefined,
        "styles": "
body {
  max-width: 1000px;
  margin: 0 auto;
  padding: 2rem;
  font-family: -apple-system, BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\",
    \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\",
    sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

code {
  font-family: source-code-pro, Menlo, Monaco, Consolas, \\"Courier New\\", monospace;
}
",
      }
    }
  />
  <h1>
    Number to words calculator
  </h1>
  <h2>
    Input
  </h2>
  <ConnectFunction />
  <h2>
    Output
  </h2>
  <ConnectFunction />
</Fragment>
`;
